src/locales.js

```diff
@@ -11,54 +11,53 @@ module.exports = {
      * Pull a translation file
      * @param  {Array}   locales Locales to pull ( by default pulls all of them )
      */
-    pull: function(locales = []){
+    pull: async function(locales = []){
+        let files;
+        try {
+            files = await this.getAllFiles();
+        } catch (err) {
+            console.log(err);
```

Comment:
Shouldn't we terminate on an error. Currently it will go forward with filtering the files that are undefined.